miden-lib/asm/miden/account.masm

```diff
@@ -8,7 +10,21 @@ export.get_id
     push.0
     # => [0]
 
-    syscall.get_account_id
+    # start padding the stack
+    push.0.0
```

Comment:
nit: why not combine these into: `push.0.0.0`? (also applies to other procedures here).